tests/unit/Table.spec.js

```diff
@@ -43,5 +43,31 @@ describe('Table', () => {
         expect(wrapper.find('caption').text()).toBe('Cars List')
       })
     })
+    
+    describe('With expandable', () => {
+      
+      it('renders only with withColumns option', () => {
+        tableProps = { ...tableProps, expandable: { withColumns: ['car_brand', 'car_model'] } }
+        const wrapper = shallowMount(Table, { propsData: tableProps })
```

Comment:
You don't need to declare the wrapper for each assertion. You can declare it in the global scope and inside each assertion, you set the props like this:
`wrapper.setProps(tableProps)`